cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -300,7 +300,7 @@ public void execute() {
             targetPath = this.sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid ballerina source path, it should either be a module name in a ballerina project or a " +
+                    "An invalid Ballerina source path. It should either be a module name in a Ballerina project or a " +
```

Comment:
Again, since we are not specifying the path, IMO this doesn't read right.

How about something like
```suggestion
                    "Invalid Ballerina source path: expected either a module name in a Ballerina project or a " +
```

Reply:
Changed to: 

```
 "Invalid Ballerina source path. Either a module name in a Ballerina project or a " +
                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension is expected.",
```